### ISSUE TITLE
example doesn't compile

### ISSUE THREAD

**Issue by geoffhendrey** [docker/libcompose#399]
if I import this "github.com/docker/libcompose/docker/ctx"
and change the first line to this:
project, err := docker.NewProject(&ctx.Context{

then it works

**Comment by joshwget** [comment]
Yeah, the example often gets overlooked. Want to put in a PR to fix it? :smile:

**Comment by vdemeester** [comment]
Ohhhh… we did not revert the example :sweat:
